dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/authorization/providers/shib/ShibUtil.java

```diff
@@ -184,8 +184,8 @@ static void mutateRequestForDevConstantHarvard1(HttpServletRequest request) {
          * Let's simulate the non-sending of displayName here.
          */
 //        request.setAttribute(displayNameAttribute, "John Harvard");
-        request.setAttribute(ShibUtil.firstNameAttribute, "John");
-        request.setAttribute(ShibUtil.lastNameAttribute, "Harvard");
+        request.setAttribute(ShibUtil.firstNameAttribute, "John2");
```

Comment:
What's the reason for a change?

Reply:
It was just to check what will happen when attributes of user that comes from institutional login would change. I will revert it to original state